README.md

```diff
@@ -352,6 +352,36 @@ CLASS_RULE_PARAM_VALIDATION = [
   - run_convert(instance, convert_to **argv) - execute the conversion for the instance with the first rule that provide the conversion (see get_convert_from_to). 'from_to' is deducted from instance. 
   - get_convert_from_to() - get the possible conversion, return [calc UUID, from, to]
 
+## CustomFilters
+This module provides the ability to define custom filters and their construction. It includes the following classes:
+* ```CustomFilterRegistryPoint```: This class is responsible for registering a specific module. 
+It should be called in the apps.py file of the respective module (see below) to enable filter construction for specific objects.
+* ```CustomFilterWizardInterface```: This interface defines the implementation details of a custom filtering wizard. 
+Each module may have different implementations for delivering the desired output.
+* ```CustomFilterWizardStorage```: This class provides a service that connects to the registered filter hub, allowing 
+retrieval of the filter construction definition for specific objects.
+
+To integrate this mechanism into an openIMIS module, follow these steps:
+* Implement the CustomFilterWizardInterface from the core module (if no custom_filters.py file exists, create one in the module) 
+to output the filter construction definition for the chosen object.
+* In the apps.py file of the module, add the following import statement: ```from core.custom_filters import CustomFilterRegistryPoint```.
+* Within the module's configuration class, in the configuration loading section, include the following lines of code:
```

Comment:
I think that those 2 steps are not align in terms of "level of complexity". First one tells user excatly how to do the import and second makes assumption that it's obvious what is configuration loading section. You can say that it has to be invoked somewhere in ready()